### ISSUE TITLE
Feature Request: Date Auto-Select

### ISSUE THREAD

**Issue by KayBeSee** [tcampb/react-calendly#74]
I am making a schedule page on my website using the Calendly embedded widget. I don't want to reveal the event details, just the calendar so that users can schedule a demo.

Here is how it looks right now:
<img width="1037" alt="Screen Shot 2021-07-06 at 4 47 51 PM" src="https://user-images.githubusercontent.com/1429113/124675433-e9cf0600-de79-11eb-884a-aae08548fe22.png">

It would be really nice to have `Date.now()` automatically selected with the times revealed like displayed below to reduce the number of clicks the user does by 1 and aesthetically look better.

<img width="1059" alt="Screen Shot 2021-07-06 at 4 47 34 PM" src="https://user-images.githubusercontent.com/1429113/124675406-de7bda80-de79-11eb-9ff9-2d50301e854c.png">

**Comment by tcampb** [comment]
Hi @KayBeSee -

You can use the `date` query param to automatically select a date so that all of the times are displayed.

Example:

https://calendly.com/acmesales/30-minute-demo?date=2021-07-08

```javascript
var dateObject = new Date(Date.now());
var month = dateObject.getMonth() + 1;
var day = dateObject.getDate();
var year = dateObject.getFullYear();

var date = [
  year,
  month < 10 ? `0${month}` : month,
  day < 10 ? `0${day}` : day
].join('-');

<InlineWidget
    url={`https://calendly.com/acmesales/30-minute-demo?date=${date}`}
/>
```

**Comment by KayBeSee** [comment]
Hey @tcampb,

Thanks for showing me how to do this! Works perfectly.

If anyone is interested, this is the final component I ended up making. It auto-expands the times depending on if the user is on mobile or desktop.

Cheers!

```
import React, { useState, useEffect } from "react";
import { InlineWidget } from "react-calendly";

const CalendlyWidget = ({ url: BASE_URL }) => {
  const [width, setWidth] = useState(0);
  function handleWindowSizeChange() {
    setWidth(window.innerWidth);
  }
  useEffect(() => {
    setWidth(window.innerWidth);
    window.addEventListener("resize", handleWindowSizeChange);
    return () => {
      window.removeEventListener("resize", handleWindowSizeChange);
    };
  }, []);

  let isDesktop = width >= 768;
  let url = BASE_URL;
  if (isDesktop) {
    var dateObject = new Date(Date.now());
    var month = dateObject.getMonth() + 1;
    var day = dateObject.getDate();
    var year = dateObject.getFullYear();

    var date = [
      year,
      month < 10 ? `0${month}` : month,
      day < 10 ? `0${day}` : day,
    ].join("-");

    url = BASE_URL.concat(`?date=${date}`);
  }

  return (
    <InlineWidget
      url={url}
      styles={{
        height: "655px",
        margin: 0,
      }}
      pageSettings={{
        hideEventTypeDetails: true,
        hideLandingPageDetails: true,
        primaryColor: "059669",
      }}
    />
  );
};

export default CalendlyWidget;
```

**Comment by tcampb** [comment]
Awesome! Thanks for sharing the code example!

Would it be helpful to add a `date` prop to the component so that you can pass in an instance of a date object instead of formatting the date string? If so, feel free to re-open this issue and I can take a look into adding this feature.

example:

```typescript
<InlineWidget
      url={url}
      date={new Date(Date.now())}
      styles={{
        height: "655px",
        margin: 0,
      }}
      pageSettings={{
        hideEventTypeDetails: true,
        hideLandingPageDetails: true,
        primaryColor: "059669",
      }}
    />
```

**Comment by KayBeSee** [comment]
I think that would be help and make the component I posted a little cleaner.

I just reopened the ticket. Thanks Tyler!

**Comment by tcampb** [comment]
Thanks for your feedback @KayBeSee! This option is now available in version `2.2.0`.

https://github.com/tcampb/react-calendly/pull/77
